2020-10-14 18:53:45,983 INFO    MainThread:28381 [wandb_init.py:_log_setup():292] Logging user logs to wandb/run-20201014_185345-gx7ht69p/logs/debug.log
2020-10-14 18:53:45,983 INFO    MainThread:28381 [wandb_init.py:_log_setup():293] Logging internal logs to wandb/run-20201014_185345-gx7ht69p/logs/debug-internal.log
2020-10-14 18:53:45,983 INFO    MainThread:28381 [wandb_setup.py:_flush():68] enabling code saving by default
2020-10-14 18:53:45,983 INFO    MainThread:28381 [wandb_setup.py:_flush():68] setting env: {'project': 'RLLIB_SWEEP_2', 'api_key': '3487a01956bf67cc7882bca2a38f70c8c95f8463', 'entity': 'ingambe', 'sweep_id': 'lh9x5rb9', 'run_id': 'gx7ht69p', 'config_paths': 'wandb/sweep-lh9x5rb9/config-gx7ht69p.yaml'}
2020-10-14 18:53:45,983 INFO    MainThread:28381 [wandb_setup.py:_flush():68] setting user settings: {'save_code': True}
2020-10-14 18:53:45,983 INFO    MainThread:28381 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2020-10-14 18:53:49,065 INFO    MainThread:28381 [wandb_run.py:_console_start():1265] atexit reg
2020-10-14 18:53:49,069 INFO    MainThread:28381 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2020-10-14 18:53:49,070 INFO    MainThread:28381 [wandb_run.py:_redirect():1138] Redirecting console.
2020-10-14 18:53:49,070 INFO    MainThread:28381 [redirect.py:install():196] install start
2020-10-14 18:53:49,070 INFO    MainThread:28381 [redirect.py:install():211] install stop
2020-10-14 18:53:49,071 INFO    MainThread:28381 [redirect.py:install():196] install start
2020-10-14 18:53:49,071 INFO    MainThread:28381 [redirect.py:install():211] install stop
2020-10-14 18:53:49,071 INFO    MainThread:28381 [wandb_run.py:_redirect():1182] Redirects installed.
2020-10-14 18:54:30,471 INFO    MainThread:28381 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 1
2020-10-14 18:54:30,472 INFO    MainThread:28381 [wandb_run.py:_restore():1210] restore
2020-10-14 18:54:30,473 INFO    MainThread:28381 [redirect.py:uninstall():215] uninstall start
2020-10-14 18:54:30,473 INFO    MainThread:28381 [redirect.py:_stop():264] _stop: stdout
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:_stop():270] _stop closed: stdout
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:_stop():276] _stop joined: stdout
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:_stop():278] _stop rd closed: stdout
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:uninstall():219] uninstall done
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:uninstall():215] uninstall start
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:_stop():264] _stop: stderr
2020-10-14 18:54:30,474 INFO    MainThread:28381 [redirect.py:_stop():270] _stop closed: stderr
2020-10-14 18:54:30,475 INFO    MainThread:28381 [redirect.py:_stop():276] _stop joined: stderr
2020-10-14 18:54:30,475 INFO    MainThread:28381 [redirect.py:_stop():278] _stop rd closed: stderr
2020-10-14 18:54:30,475 INFO    MainThread:28381 [redirect.py:uninstall():219] uninstall done
2020-10-14 18:54:30,484 INFO    MainThread:28381 [wandb_run.py:_restore():1210] restore
2020-10-14 18:54:33,367 ERROR   MainThread:28381 [wandb_run.py:_atexit_cleanup():1256] Problem finishing run
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1246, in _atexit_cleanup
    self._on_finish()
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1383, in _on_finish
    wandb.termlog(status_str)
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/errors/term.py", line 33, in termlog
    click.echo(line, file=sys.stderr, nl=newline)
  File "/root/miniconda3/lib/python3.8/site-packages/click/utils.py", line 273, in echo
    file.flush()
OSError: [Errno 5] Input/output error
2020-10-14 18:54:33,374 INFO    MainThread:28381 [internal.py:handle_exit():137] Internal process exited
